app/controllers/admin/rdvs_collectifs_controller.rb

```diff
@@ -30,9 +30,38 @@ def create
     end
   end
 
+  def edit_users
+    @rdv = Rdv.find(params[:id])
+
+    @add_user_ids = params[:add_user].to_a + params[:user_ids].to_a
+    users_to_add = User.where(id: @add_user_ids)
+    @rdv_users_to_add = users_to_add.ids.map { @rdv.rdvs_users.build(user_id: _1) }
+
+    authorize(@rdv, :edit?)
```

Comment:
Pinaillage: on peut mettre le authorize juste après la ligne `@rdv =`? J’ai l’impression que c’est là que je m’attends à la trouver.
```suggestion
    authorize(@rdv, :edit?)

    @add_user_ids = params[:add_user].to_a + params[:user_ids].to_a
    users_to_add = User.where(id: @add_user_ids)
    @rdv_users_to_add = users_to_add.ids.map { @rdv.rdvs_users.build(user_id: _1) }
```